src/plugins/data/common/search/tabify/response_writer.test.ts

```diff
@@ -174,19 +184,23 @@ describe('TabbedAggResponseWriter class', () => {
         });
 
         expect(response.columns[1]).toHaveProperty('id', 'col-1-2');
-        expect(response.columns[1]).toHaveProperty('name', 'Count');
+        expect(response.columns[1]).toHaveProperty('name', 'Unique count of machine.os.raw');
         expect(response.columns[1]).toHaveProperty('meta', {
           index: 'logstash-*',
+          field: 'machine.os.raw',
           params: {
             id: 'number',
           },
           source: 'esaggs',
           sourceParams: {
+            appliedTimeRange: undefined,
```

Comment:
Looks like this is added because `count` is handled in a unique way, but most metrics have this?